smart-contracts/assembly/contracts/NFT/NFT.ts

```diff
@@ -255,7 +258,6 @@ export function nft1_mint(_args: StaticArray<u8>): void {
     .nextString()
     .expect('mintAddress argument is missing or invalid');
 
-  // TODO:  Check Address validity
   _increment();
 
   const tokenToMint = _currentSupply().toString();
```

Comment:
Increment set the "counterkey" and currentSupply read it. its not optimal.
_increment should return the setted value, then you can directly use it for `tokenToMint`